app/main/views/direct_award.py

```diff
@@ -0,0 +1,193 @@
+from flask import jsonify, abort, current_app, request
+from sqlalchemy.exc import IntegrityError
+from sqlalchemy import desc
+
+from dmapiclient.audit import AuditTypes
+from .. import main
+from ... import db
+from ...models import User, DirectAwardProject, AuditEvent, DirectAwardSearch
+from ...utils import (
+    get_json_from_request, get_int_or_400, json_has_required_keys, pagination_links,
+    get_valid_page_or_1, validate_and_return_updater_request, validate_user_can_access_direct_award_project_or_403)
+
+
+@main.route('/direct-award/projects', methods=['GET'])
+def list_projects():
+    user_id = get_int_or_400(request.args, 'user_id')
+    if user_id is None:
+        abort(400, "User ID not supplied")
+
+    page = get_valid_page_or_1()
+
+    projects = DirectAwardProject.query.filter(DirectAwardProject.users.any(id=user_id))
+    projects = projects.order_by(desc(DirectAwardProject.created_at), desc(DirectAwardProject.id))
```

Comment:
Usually `asc` order by id is preferred as the default sort since it allows us to safely iterate through API pages without catching duplicates (since for desc order someone creating a new project between page=1 request and page=2 request will shift 100th record to 101, making it appear twice).

Are we planning a paginated display of all projects on the frontend? If we do, this should be a "reverse order", similar to "human" flag in briefs or "earliest_first" in audit events.